gratipay/billing/exchanges.py

```diff
@@ -183,14 +184,31 @@ def create_card_hold(db, participant, amount):
     msg = "Holding " + amount_str + " on Balanced for " + username + " ... "
 
     hold = None
+    error = ""
     try:
-        card = thing_from_href('cards', route.address)
-        hold = card.hold( amount=cents
-                        , description=username
-                        , meta=dict(participant_id=participant.id, state='new')
-                         )
-        log(msg + "succeeded.")
-        error = ""
+        result = braintree.Transaction.sale({
+            'amount': str(cents/100.0),
```

Comment:
You positive? I thought it was a `Decimal` here. Regardless, if Braintree accepts dollars we should just pass dollars. We only convert to cents in `_prep_hit` because Balanced wanted cents (though does `ach_credit` also use `_prep_hit`?